README.md

```diff
@@ -50,8 +50,19 @@ to provide the best of both worlds.
         1. ***void fetchAd(AdRetriever retriever, RetrieveAdCallback callback)*** 
         2. ***void fetchQuePoint(CuePointsRetriever retriever, CuePointCallBack callBack)***
         
-
-
+    5. If you want to quickly test the pre-roll and mid-roll video ads using the default components, just simply change 1 line of code in [PlayerModuleDefault](lib/src/main/java/com/tubitv/media/di/PlayerModuleDefault.java#L131): <br/>  
+    ```java
+  
+       @Override
+       public void fetchQuePoint(CuePointsRetriever retriever, CuePointCallBack callBack) {
+    
+           callBack.onCuePointReceived(new long[] { 60000, 900000, 1800000, 3600000 });
+                //"AdBreak point at 0s, 1min, 15min, 30min, 60min. With each adbreak showing one ads"
+       }
+   
+    ```
+    Then, run the app, and see the testing pre-roll and mid-roll ads in demo.
```

Comment:
Why "in demo"?  Is this a demo app?  I thought this would be users integrating into their own app.

Reply:
there is a lib module that user can import to their project, and there is app in this project that use as demo. Maybe i should also point it out in the documentation?

Reply:
I'm worried "run the app" is too vague.  How about we modify it to "Then, run your app (or the demo app), to see the test pre-roll and mid-roll ads at each specified CuePoint."